### DISCUSSION TITLE
Schlummer-Timer

### DISCUSSION THREAD

**Discussion by ChrisH172** [beabel/radiowecker#66] [Fragen & Antworten / Q&A] [answered]
Bei mir läuft der Schlummer-Timer nach der halben eingestellten Zeit ab. Habt Ihr auch das Problem? Ursache könnte sein, dass der µC etwas zu schnell läuft und somit schon nach 59 sec und dann nochmals (somit 2x pro Minute) die Schlummer-Minuten (snoozeWait) dekrementiert.

Abhilfe: Beim Start des Schlummer-Timers wird mit millis() eine Ablaufzeit berechnet. Im Anhang habe ich die Änderung abgelegt (Basis: V3.0.5).

Die zuvor beschriebenen Änderungen im Forum "Uhrzeit stimmt nicht immer" (if ((millis() - tick) > 60000) auf 30000 setzen) verkürzen die Ablaufzeit nochmals. Durch die Abhilfe wird dieses Problem umgangen.

Wie ist Eure Meinung?

[radiowecker.ino.txt](https://github.com/user-attachments/files/16935387/radiowecker.ino.txt)
[tft_display.ino.txt](https://github.com/user-attachments/files/16935388/tft_display.ino.txt)
[webserver.ino.txt](https://github.com/user-attachments/files/16935389/webserver.ino.txt)
[00_settings.h.txt](https://github.com/user-attachments/files/16935390/00_settings.h.txt)

**Comment by beabel** [comment, marked as answer]
Das Problem sollte ab Version 3.0.6 gelöst sein.

Thx an @ChrisH172
